PyFile/SavePyFile.py

```diff
@@ -65,8 +64,8 @@ def canOwnSubjectHierarchyItem(self, itemID):
             if mimetype == "text/x-python" or fileType == "python":
                 return 1.
 
-        else:
-            print("No node found for itemID:", itemID)
+        # else:
```

Comment:
Just a note, it's usually good practice not to leave "dead code" in the source files unless it's really something that could be useful for future debugging.